containers/kubernetes/how-to/upgrade-kubernetes-version.mdx

```diff
@@ -61,17 +61,21 @@ scw k8s version list
 
 You should get an output similar to the following, providing a list of all relevant components:
 ```
-1.25.9   [cilium calico kilo]                [none]                 [containerd]
-1.24.13  [cilium calico weave flannel kilo]  [none]                 [containerd crio]
-1.23.17  [cilium calico weave flannel kilo]  [none nginx traefik2]  [containerd crio docker]
+NAME     AVAILABLE CNIS                      AVAILABLE CONTAINER RUNTIMES
+1.28.2   [cilium calico kilo]                [containerd]
+1.27.6   [cilium calico kilo]                [containerd]
+1.26.9   [cilium calico kilo]                [containerd]
+1.25.14  [cilium calico kilo]                [containerd]
+1.24.17  [cilium calico weave flannel kilo]  [containerd]
 ```
 
-### Ingress controllers
-
-Managed ingress controllers have been deprecated since the minor 1.24 release. As a result, we provide the [Easy Deploy feature](/containers/kubernetes/how-to/enable-easy-deploy/) instead to set up an ingress controller.
-You can also deploy an ingress controller by yourself and fine-tune its settings according to your needs.
+### CNIs
 
-Run the `scw k8s cluster update $CLUSTER_ID ingress=none` command to update a cluster to a version superior to 1.23. It is necessary to deactivate the managed ingress controller.
+If your cluster is using a deprecated CNI, and you are willing to upgrade to a newer k8s version, you will most likely have to spin up a new Kapsule cluster.
+As of now, we do not provide an easy way to change CNI. This component is integrated within each cluster node. This results in a tricky transition.
+For more help, check out the following resources:
```

Comment:
```suggestion
If your cluster currently uses a deprecated Container Network Interface (CNI) and you are willing to upgrade to a more recent Kubernetes version, the recommended approach is to create a new Kapsule cluster.
Unfortunately, at present, there isn't a straightforward method to modify the CNI since it is tightly integrated within each cluster node, making the transition a complex process.
For more help, check out the following resources:
```